scripts/cocktailsToMakeModel.js

```diff
@@ -0,0 +1,54 @@
+var cocktailsToMake = {
+  ingredientString: '',
+  paramString: '',
+  cocktailObjects: [],
+  listTemplate: '',
+
+  createIngredientString: function (){
+    $('#addedIngredients li').each(function(index){
+      cocktailsToMake.ingredientString += ('"' + $(this).text() + '"' + ', ');
```

Comment:
Instead of getting the text of the ingredients and then querying the database for what the ID of each of these ingredients is, you should do the following: 

When the ingredient is picked from the autocomplete, add the following html that has the ID of the element that the user has into the DOM: 
"<li data-id="53">vodka</li>"

This way, you can just query for $('#addedIngredients li') and have the ID's right away, without requiring an additional query at the end.